libvirt/resource_libvirt_domain.go

```diff
@@ -256,6 +256,9 @@ func resourceLibvirtDomainCreate(d *schema.ResourceData, meta interface{}) error
 		domainDef.Devices.Console = consoles
 	}
 
+	controller := defController{Type:"scsi", Model:"virtio-scsi"}
+	domainDef.Devices.Controller = append(domainDef.Devices.Controller, controller)
```

Comment:
I see you are adding this controller by default, and I was wondering what happens when the disk is not SCSI...